pkgs/development/compilers/dotnet/default.nix

```diff
@@ -70,6 +70,11 @@ let
       // {
         inherit callPackage fetchNupkg buildDotnetSdk;
 
+        dotnet-buildsdk-input = writeScript "dotnet-buildsdk-input.sh" (
```

Comment:
Sorry, I didn't notice this the first time, but it should be `writeScriptBin`/`writeShellScriptBin` instead of `writeScript`.

There shouldn't be any extensions either as well, otherwise people will need to specify it when running it.

Reply:
done